php/example_code/secretsmanager/ListSecrets.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ *  ABOUT THIS PHP SAMPLE: This sample is part of the KMS Developer Guide topic at
+ *  https://docs.aws.amazon.com/kms/latest/developerguide/programming-key-policies.html
+ *
+ */
+
+require 'vendor/autoload.php';
+
+use Aws\SecretsManager\SecretsManagerClient;
+use Aws\Exception\AwsException;
+
+/**
+ * List the SecretId for each Secret in the account for the selected region.
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+
+//Create a Secrets Manager Client
+$client = new SecretsManagerClient([
+    'profile' => 'default',
+    'version' => '2017-10-17',
+    'region' => 'us-west-2'
+]);
+
+try {
+    $result = $client->listSecrets([

```

Comment:
This can all be in a single row since there are no parameters, i.e. `$result = $client->listSecrets([]);`